config/environment.js

```diff
@@ -126,6 +126,7 @@ module.exports = function(environment) {
     etagAPI: process.env.ETAG_API,
     publisherAPI: process.env.PUBLISHER_API,
     platformEventsAPI: process.env.PLATFORM_EVENTS_SERVICE,
+    gothamistStories: process.env.GOTHAMIST_STORIES,
     wnycDonateURL: 'https://pledge3.wnyc.org/epledge/main?ref=button-donate-header',
```

Comment:
I realized recently after examining that ember scheduling app they showed at ember conf that these types of variables should really go under `config.APP`? Not sure if we should just keep doing it the way we do, start putting them in under `config.APP` as the new way, or bother to move them all. Pretty minor, but just a thought.